notifications/detekt.yml

```diff
@@ -26,3 +26,9 @@ style:
   ReturnCount:
     active: true
     max: 10
+
+complexity:
+  TooManyFunctions:
+    excludes: ["**/util/**", "**/test/**"]
```

Comment:
I have split up helper functions in PR #31 
I think it is better not to add generic exceptions to rules